src/System.Data.Common/tests/System/Data/DataSetTest.cs

```diff
@@ -1566,7 +1562,7 @@ public void WriteXmlModeSchema()
         [Fact]
         public void WriteXmlModeSchema1()
         {
-            RemoteExecutorForUap.Invoke(() =>
+            // Keeping the brackets as the test otherwise starts to fail.
```

Comment:
Do we know why?

Reply:
That does not make sense. The IL will be identical regardless of whether these brackets are here in the source.

Reply:
This is about removing the indenting which caused the test to fail as the test input might be encoded differently. I reverted my change to not touch the test data at all.